algorithms/CPlusPlus/Arrays/specialindex2.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+@author: nandinisahu407
+special index-> if after deleting element from index i , sum of even index=sum of odd index
+approach->
+after deleting ,previous element at odd index will be now at even index and vice versa
+s_odd= odd[0 to i]+ even[i+1 to len]
+s_even=even[0 to i]+odd[i+1 to len]
+
+*/
+#include<iostream>
+using namespace std;
+ 
+int main(){
+    int num;
+    cout<<"enter length"<<endl;
+    cin>>num;
+    int arr[num];

```

Comment:
Why not use a much more safe and reliable stl container instead of using a VLA ( variable length array ) here and relying on the particular C++ implementation to support it as a non-standard extension ?